inc/projecttask.class.php

```diff
@@ -1299,11 +1299,22 @@ static function populatePlanning($options=array()) {
                      AND ";
       }
 
+      $DONE_EVENTS = '';
+      if (!$options['display_done_events']) {
+         $DONE_EVENTS = "`glpi_projecttasks`.`percent_done` < 100
+                         AND (glpi_projectstates.is_finished = 0
+                              OR glpi_projectstates.is_finished IS NULL)
+                         AND ";
+      }
+
       $query = "SELECT `glpi_projecttasks`.*
                 FROM `glpi_projecttaskteams`
                 INNER JOIN `glpi_projecttasks`
                   ON (`glpi_projecttasks`.`id` = `glpi_projecttaskteams`.`projecttasks_id`)
+                LEFT JOIN `glpi_projectstates`
```

Comment:
Should not the join be conditionned as well? Well, not really important, but this could maybe save query time when the option is on.